website/content/en/highlights/2023-08-30.md

```diff
@@ -0,0 +1,33 @@
+---
+date: "2023-08-30"
+title: "YAML default configuration format"
+description: "YAML is now the default configuration format"
+authors: ["pront"]
+pr_numbers: [18325, 18345, 18378, 18388, 18420]
+release: "0.33.0"
+hide_on_release_notes: false
+badges:
+  type: announcement
+---
+
+## What is changing?
+
+We are thrilled to announce that [YAML](https://yaml.org/) is now the default Vector configuration format.
+
+YAML is unambiguous, offers great readability due to its clean syntax and is a widely adopted configuration language. In addition, many tools support YAML out of the box.
+
+Existing TOML and JSON configurations are not affected by this decision and will work as usual.
```

Comment:
```suggestion
## What is changing?

We are happy to announce that the default configuration language for Vector has been updated from [TOML](https://toml.io/en/) to [YAML](https://yaml.org/).

We hope that this will improve the overall clarity of Vector configurations as we've found that TOML configurations quickly become difficult to read when they contain more than a small number of components. Updating the default in the documentation and CLI defaults will encourage users to start out with YAML rather than switching only once they outgrow TOML. This also aligns the default configuration language with the required language, YAML, when deploying via Helm, thus reducing some friction.

Existing TOML and JSON configurations are not affected by this decision and will work as usual.
```